docs/source/basic_tutorials/using_cli.md

```diff
@@ -0,0 +1,55 @@
+# Using TGI through CLI
+
+You can use CLI tools of TGI to download weights, serve and quantize models, or get information on serving parameters. 
+
+## Installing TGI for CLI
```

Comment:
All this section I would move to a new `installation.md` section as it's done for other libraries (e.g. https://huggingface.co/docs/accelerate/basic_tutorials/install )